src/validaze/core.clj

```diff
@@ -628,7 +627,7 @@
           (validator-fn vec-or-single))]
     (if (empty? errors)
       (success-fn vec-or-single)
-      errors)))
+      (into [] errors))))
```

Comment:
This is wrong. Needs to not happen on associative data structures but rather only on sequential ones.